tsconfig.json

```diff
@@ -18,8 +19,19 @@
       "node",
       "jasmine",
       "fontfaceobserver"
-    ]
+    ],
+    "baseUrl": ".",
+    "paths": {
+      "@blackbaud/skyux-builder/*": [
+        "./node_modules/@blackbaud/skyux-builder/*",
+        "./*"
+      ]
+    }
   },
+  "exclude": [
+    "node_modules",
+    "**/*.aot.ts"
```

Comment:
Ignore TS's `Cannot find module './ngfactory/app/app.module.ngfactory'.` within the **main-internal.aot.ts** file.